docker-compose.yml

```diff
@@ -1,20 +1,24 @@
-db:
-  image: mysql:5.6
-  environment:
-    - MYSQL_DATABASE=mapknitter
-    - MYSQL_USER=mapknitter
-    - MYSQL_PASSWORD=mapknitter
-    - MYSQL_RANDOM_ROOT_PASSWORD=true
-  volumes:
-    - /srv/mapknitter_priv1/mk1_database:/var/lib/mysql
-web:
-  build: .
-  command: /bin/bash -c "sleep 5 && bundle exec rails s -p 3000 -b '0.0.0.0'"
-  environment:
-    - RAILS_ENV=production
-  volumes:
-    - ./:/app
-  ports:
-    - "127.0.0.1:3000:3000"
-  links:
```

Comment:
Using depends_on since its a stronger relation is required for app/database

Reply:
Yes